specs/subresourceintegrity/spec.markdown

```diff
@@ -595,15 +577,15 @@ failed resource with a different one.
 
 Whenever a user agent attempts to [obtain a resource][] pointed to by a
 `link` element that has a `rel` attribute with the value of `stylesheet`,
-insert the following step between steps 1 and 2:
+modify step 4 to read:
 
-1.  Set the [integrity metadata][] of the request to the value
-    of the element's `integrity` attribute.
-{:start="2"}
+Do a potentially CORS-enabled fetch of the resulting absolute URL, with the
+mode being the current state of the element's crossorigin content attribute,
```

Comment:
As a clarification, this is an exact copy of the text already present in http://www.w3.org/TR/html5/document-metadata.html#concept-link-obtain, just extended with ", and the [integrity metadata][] of the request to the
+value of the element's `integrity` attribute", so really the only part of this up for debate is that last clause :-)

And, yes, this still requires CORS because the "match" algorithm still does the "Is resource eligible for integrity validation check," which will return "false" if resource is not same origin or CORS enabled.